kontrakter/src/main/kotlin/no/nav/pleiepengerbarn/uttak/kontrakter/Uttaksgrunnlag.kt

```diff
@@ -54,3 +56,11 @@ data class UtenlandsoppholdInfo(
     @JsonProperty("utenlandsoppholdÅrsak") val utenlandsoppholdÅrsak: UtenlandsoppholdÅrsak,
     @JsonProperty("landkode") val landkode: String?
 )
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+@JsonAutoDetect(getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, fieldVisibility = JsonAutoDetect.Visibility.ANY)
+data class OverstyrtInput(
+    @JsonProperty("overstyrtUttaksgrad") val overstyrtUttaksgrad: BigDecimal,
```

Comment:
Det kan være at vi på sikt vil ønske å legge inn mulighet for overstyring av normalarbeidstid og faktisk arbeidstid, men det burde være mulig å legge til senere. Ved en slik endring kan "overstyrtUttaksgrad" enten brukes til å regne ut faktisk arbeidstid ... eller så kan faktisk arbeidstid være et eget felt.

Reply:
...så tenker det er greit å ikke komplisere det nå.